packages/frontend/src/infrastructure/HttpService/HttpService.ts

```diff
@@ -1,4 +1,4 @@
-import { TASK } from 'shared'
+import { TASK } from 'packages/shared/src/paths/task'
```

Comment:
you should be able to import it just from 'shared'

Reply:
I agree, if there is an problem, we can try to work it out